src/workspace/mod.rs

```diff
@@ -3089,6 +3169,20 @@ pub mod test {
           "proseWrap": "always",
           "singleQuote": true,
           "semiColons": true,
+          "quoteProps": "consistent",
+          "newLineKind": "lf",
+          "useBraces": "always",
+          "bracePosition": "nextLine",
+          "singleBodyPosition": "maintain",
+          "nextControlFlowPosition": "maintain",
+          "trailingCommas": "onlyMultiLine",
+          "operatorPosition": "nextLine",
+          "jsx.bracketPosition": "nextLine",
+          "jsx.forceNewLinesSurroundingContent": true,
+          "jsx.multiLineParens": "always",
+          "typeLiteral.separatorKind": "semiColon",
+          "spaceAround": true,
+          "spaceSurroundingProperties": true,
```

Comment:
Do these settings apply to all the languages or just JS/TS? (besides the ones starting with `jsx.` as these are pretty self-explanatory).

I think that this is an ample occasion to make sure we future-proof formatting options and not make languages other than JS/TS an after-thought in term of configuration and avoid potential clashes in the future.

Reply:
I suppose these only apply to JS and TS.

> I think that this is an ample occasion to make sure we future-proof formatting options and not make languages other than JS/TS an after-thought in term of configuration and avoid potential clashes in the future.

How about using top-level namespace for JS and TS as they are primary target of Deno, and prefixing others with language names like `css.quotes`, `jsonc.trailingComma`, etc?

Reply:
Yeah, that sounds good to me